### PR TITLE
E2E: movie tunnel : create a movie - main information to production information

### PR DESCRIPTION
**Pull request by n-germain-work**
State issue number in your PR to close it when PR merged: issue #XXX
e.g. : "Resolve the bundle size ❤️: issue #1234"

### CODE REVIEW

**Comment by phpgeekfr** [comment]
as a second step, maybe the same test can be done on catalog also and check that the production statuses are are not all available ?
image

File: apps/festival/festival-e2e/src/test being reworked/movie-tunnel.spec.ts [hunk 1]
```diff
@@ -0,0 +1,345 @@
+﻿/// <reference types="cypress" />
```
Comment (phpgeekfr) [inline apps/festival/festival-e2e/src/test being reworked/movie-tunnel.spec.ts:1]: this one will be removed ?

File: apps/festival/festival-e2e/src/integration/dashboard/movie-tunnel.spec.ts [hunk 2]
```diff
@@ -1,345 +1,163 @@
...
+    getInList('option_', memberStatus[movie.directors[0].status]);
+    get('director-category').click();
+    getInList('option_', directorCategory[movie.directors[0].category]);
+    get('director-desc').type(movie.directors[0].description);
+    get('director-film-title').eq(0).type(movie.directors[0].filmography[0].title);
+    get('director-film-year').eq(0).type(movie.directors[0].filmography[0].year.toString());
+    get('table-save').click();
+    get('next').click();
+
+    //storyline elements
```
Comment (phpgeekfr) [inline apps/festival/festival-e2e/src/integration/dashboard/movie-tunnel.spec.ts:139]: is this useful ?

File: apps/festival/festival-e2e/src/integration/dashboard/movie-tunnel.spec.ts [hunk 3]
```diff
@@ -1,345 +1,163 @@
...
+} from '@blockframes/model';
+import { user, org, permissions, inDevelopmentMovie } from '../../fixtures/dashboard/movie-tunnel';
+
+const injectedData = {
+  [`users/${user.uid}`]: user,
+  [`orgs/${org.id}`]: org,
+  [`permissions/${permissions.id}`]: permissions,
+};
+
+describe('Screenings', () => {
```
Comment (phpgeekfr) [inline apps/festival/festival-e2e/src/integration/dashboard/movie-tunnel.spec.ts:31]: bad describe text

**Review by phpgeekfr — APPROVED** [review]

File: apps/festival/festival-e2e/src/test being reworked/movie-tunnel.spec.ts (hunk 1 above)
Comment (n-germain-work) [reply apps/festival/festival-e2e/src/test being reworked/movie-tunnel.spec.ts:1]: yes, when all its logic (and more) will be in place. I could still use it to be sure not to forget anything.

File: apps/festival/festival-e2e/src/integration/dashboard/movie-tunnel.spec.ts (hunk 2 above)
Comment (n-germain-work) [reply apps/festival/festival-e2e/src/integration/dashboard/movie-tunnel.spec.ts:139]: I believe so yes, these tests are going to be super long ones, easier for later maintenance.

**Pull request merged**